pagopa/io-app
# feat(IT Wallet): [SIW-258] Fix iOS build

This reverts commit ff208e7bae036631f014648e06496c1fd87e0072.

## Short description
This PR removes some unwanted changes commited when testing fastlane locally.

## How to test
`cd ios && bundler exec pod install && cd .. && yarn run-ios` should not yield any error.
